src/hii_landuse.py

```diff
@@ -26,70 +22,53 @@ class HIILanduse(EETask):
 
-    def FUNC_IC(item,image_coll):
-        year_property = ee.Number(item).add(1992)
-        single_year = ee.ImageCollection([ESACCI.select([item]).set('year',year_property)])
-        return ee.ImageCollection(image_coll).merge(single_year)
-  
-
-
-
     gpw_cadence = 5
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.set_aoi_from_ee("{}/sumatra_poc_aoi".format(self.ee_rootdir))
 
-    #def test_function(number):
-    #    print(number)
-
-    #print(test_function(5555555555555))
-
-    def square(x):
-      square = x * x
-      return square
 
-    test1 = 'test1'
-
-    def something(self):
-      something_else = self.test1
-      return something_else
 
     def calc(self):
-        print(self.test1) # works
-        print(self.something()) # works
+        ESACCI = ee.Image(self.inputs['esacci']['ee_path']).select('b24')
+        gpw_2015 = ee.Image(self.inputs['gpw_2015']['ee_path'])
+
+        caspian = ee.FeatureCollection(self.inputs['caspian']['ee_path'])
 
-        ESACCI = ee.ImageCollection(self.inputs['esacci']['ee_path'])
-        year_list =   ee.List.sequence(0,23)
-        ESACCI_ic =   ee.ImageCollection(year_list.iterate(FUNC_IC(), ee.ImageCollection([])))
+        jrc = ee.Image(self.inputs['jrc']['ee_path'])\
+                        .select('occurrence')\
+                        .lte(75)\
+                        .unmask(1)\
+                        .multiply(ee.Image(0).clip(caspian).unmask(1))
 
-        print(year_list.getInfo())
+        ocean = ee.Image(self.inputs['ocean']['ee_path'])
 
 
 
 
+        gpw_300m = gpw_2015.resample().reproject(crs='EPSG:4326',scale=300)
+        popdens_oneplus = gpw_300m.gte(1)
 
-        gpw_2015 = ee.ImageCollection(self.inputs['gpw_2015']['ee_path'])
+        ESACCI_oneyear_postpopdens = ESACCI.multiply(popdens_oneplus.multiply(10))
 
-        ee_taskdate = ee.Date(self.taskdate.strftime(self.DATE_FORMAT))
-        gpw_prior = gpw.filterDate(ee_taskdate.advance(-self.gpw_cadence, 'year'), ee_taskdate).first()
-        gpw_later = gpw.filterDate(ee_taskdate, ee_taskdate.advance(self.gpw_cadence, 'year')).first()
-        gpw_diff = gpw_later.subtract(gpw_prior)
-        numerator = ee_taskdate.difference(gpw_prior.date(), 'day')
-        gpw_diff_fraction = gpw_diff.multiply(numerator.divide(self.gpw_cadence * 365))
-        gpw_taskdate = gpw_prior.add(gpw_diff_fraction)
-        gpw_taskdate_300m = gpw_taskdate.resample().reproject(crs=self.crs, scale=self.scale)
 
-        gpw_venter = gpw_taskdate_300m.add(ee.Image(1))\
-            .log()\
-            .multiply(ee.Image(3.333))
-        # TODO: mask water with centralized HII-defined water images
+        ESACCI_remap_input =  [10,11,12,20,30,40,190,1000,1100,1200,1210,1220,1300]
+        ESACCI_remap_output = [ 7, 7, 7, 8, 6, 4, 10,   4,   4,   4,   4,   4,   4]
+        ESACCI_remapped = ESACCI_oneyear_postpopdens.remap(ESACCI_remap_input,ESACCI_remap_output,0)\
```

Comment:
See above -- it looks to me like this task basically takes a precalculated LC image and classifies the areas with people. Seems to me we need to start further upstream if possible. If this is just a placeholder, that's fine, just make sure it's clear in the cards what else needs to be done.